server/routes/charts.js

```diff
@@ -1,5 +1,5 @@
 import { Router } from 'express'
-import { ObjectID } from 'mongodb'
+import { ObjectId } from 'mongodb'
```

Comment:
Does the new `mongodb` library no longer have `ObjectID` as spelt before? @Elkrival

Reply:
Yes ObjectID is deprecated and now it is ObjectId